backend/metasfresh-webui-api/src/main/java/de/metas/ui/web/accounting/filters/FactAcctFilterConverter.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * #%L
+ * metasfresh-webui-api
+ * %%
+ * Copyright (C) 2020 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.ui.web.accounting.filters;
+
+import com.google.common.collect.ImmutableSet;
+import com.jgoodies.common.base.Objects;
+import de.metas.acct.api.IElementValueDAO;
+import de.metas.acct.api.impl.ElementValueId;
+import de.metas.ui.web.document.filter.DocumentFilter;
+import de.metas.ui.web.document.filter.sql.SqlDocumentFilterConverter;
+import de.metas.ui.web.document.filter.sql.SqlDocumentFilterConverterContext;
+import de.metas.ui.web.document.filter.sql.SqlParamsCollector;
+import de.metas.ui.web.window.model.sql.SqlOptions;
+import de.metas.util.Services;
+import lombok.NonNull;
+import org.adempiere.exceptions.FillMandatoryException;
+import org.compiere.model.I_Fact_Acct;
+import org.compiere.util.DB;
+
+public class FactAcctFilterConverter implements SqlDocumentFilterConverter
+{
+	public static final transient FactAcctFilterConverter instance = new FactAcctFilterConverter();
+
+	static final String FILTER_ID = "fact-acct";
```

Comment:
i would use `accountId` instead of  `fact-acct`.

Reply:
done

Reply:
done